### PR TITLE
CRUD Generator: Add support for enum arrays in input

### PR DESCRIPTION
**Pull request by nsams**

### CODE REVIEW

File: .changeset/silver-dolphins-hide.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-api": minor
+---
+
+crud generator: add support for enum arrays in input
```
Comment (johnnyomair) [inline .changeset/silver-dolphins-hide.md:5]: IMO we should start being consistent with the name of the generator. API Generator, API CRUD Generator, CRUD Generator, CRUD Generator (API)... What do you prefer?

File: packages/api/cms-api/src/generator/generate-crud-input.ts [hunk 2]
```diff
@@ -84,6 +84,19 @@ export async function generateCrudInput(
             decorators.push(`@IsEnum(${enumName})`);
             decorators.push(`@Field(() => ${enumName}, ${fieldOptions})`);
             type = enumName;
+        } else if (prop.type === "EnumArrayType") {
+            if (prop.nullable) {
+                console.warn(`${prop.name}: Nullable enum arrays are not supported`);
```
Comment (johnnyomair) [inline packages/api/cms-api/src/generator/generate-crud-input.ts:89]: Why don't we support this? Is this the same for other array types?

**Review by johnnyomair — COMMENTED** [review]
What about the Admin CRUD Generator? Do we even support normal enums?

File: .changeset/silver-dolphins-hide.md (hunk 1 above)
Comment (nsams) [reply .changeset/silver-dolphins-hide.md:5]: the changeset is in the context of the api package, that's why I did leave out the "api" (yes, I did actually think about that)

Comment (johnnyomair) [inline .changeset/silver-dolphins-hide.md:5]:
```suggestion
CRUD Generator: Add support for enum arrays in input
```

**Review by johnnyomair — APPROVED** [review]

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**